conf/environment.yml

```diff
@@ -1,8 +1,10 @@
 name: yosys-env
 channels:
+  - conda-forge
```

Comment:
conda-forge is needed for some surelog dependency?

Reply:
Indeed, that's the case.